docs/performance/inserts/methods.rst

```diff
@@ -123,6 +126,15 @@ For example, using the CrateDB Python client, the following is possible:
 Here, you can vary the number of rows being inserted without having to change
 the prepared statement.
 
+.. warning::
+    When inserting using ``UNNEST``, e.g. using the Python client, rows that
+    produce errors while inserting may be dropped without an error message
+    being returned. This can produce inconsistencies and unexpected results.
+    This happens when the ``SELECT`` using ``UNNEST`` involves rows with
```

Comment:
perhaps this example/explanation should be moved above the concluding statement about the behavior producing inconsistencies (to improve narrative flow)?